src/commands/create.js

```diff
@@ -193,6 +194,25 @@ export default asyncCommand({
 			spinner.succeed('Done!\n');
 		}
 
+		if (argv.less || argv.sass || argv.stylus) {
+			const { less, sass, stylus } = argv;
+			const srcDir = path.resolve(target, 'src');
+
+			let extension;
+
+			if (less) extension = '.less';
+			if (sass) extension = '.scss';
+			if (stylus) extension = '.styl';
+
+			await copy(srcDir, srcDir, {
```

Comment:
I'm fine adding `glob` - we'll probably want to use that module elsewhere anyway :)

@reznord I had initially looked at `renamer` too, there is a node counterpart for it - the problem was that it's synchronous.